### ISSUE TITLE
Not able to create "New Issues" - Safari only.

### ISSUE THREAD

**Issue by jxsl13** [gogits/gogs#4113]
I could not exactly find this issue, so I hope it's not a double one.

I am running Gogs on a raspberry pi 2, runs greatly.
So I tried to create a new issue on a cloned project, somehow could not.
Thoughts that only I have this problem, running most recent build, so I checked out my problem on my test account @  [try.gogs.io/jxsl13/info-a](https://try.gogs.io/jxsl13/info-a) and I have the same problem there. 
As I am using Safari, I thought tat my browser was the problem and it seems to be the case.
After trying to create a new issue with Opera, it worked.
[youtube](https://www.youtube.com/watch?v=ZPbqwO0TvtU&feature=youtu.be)
Even after waiting for a while, it seems not to load the JavaScript( or whatever is used).
Took a look at my settings, JavaScript is enabled, tho.

***Can be closed ._. : somehow the interface gets blocked by my ad blocker.***
- Gogs version (or commit ref): 
- Git version: 
- Operating system: 
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gogs.io:
  - [x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:
